src/MissedCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Card and MissedCard components are very similar, and the difference between the two could be taken care of with styling.   Not sure this justifies them being separate components.